org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opent2t-translator-com-smartthings-binaryswitch",
-  "version": "1.0.0",
+  "version": "0.2.0",
```

Comment:
We never published v1.0.0 of this translator, the latest on NPM is 0.0.1 so we can move to 0.2.0 to represent the new resource model.

---
In reply to: [90066887](https://github.com/openT2T/translators/pull/130#discussion_r90066887) [](ancestors = 90066887)